packages/api/src/utils.js

```diff
@@ -33,23 +35,48 @@ const decodeStateTransition = async (client, base64) => {
 
   switch (decoded.type) {
     case StateTransitionEnum.DATA_CONTRACT_CREATE: {
+      const dataContractConfig = stateTransition.getDataContract().getConfig()
+
+      decoded.internalConfig = {
+        canBeDeleted: dataContractConfig.canBeDeleted ?? null,
```

Comment:
only requires_identity_encryption_bounded_key and requires_identity_decryption_bounded_key are optional

Reply:
✅